loki/transformations/single_column/tests/test_scc.py

```diff
@@ -338,7 +338,10 @@ def test_scc_annotate_openacc(frontend, horizontal, blocking, acc_data):
         if acc_data:
             assert driver_loops[0].pragma[0].content == 'parallel loop gang vector_length(nlon)'
         else:
-            assert driver_loops[0].pragma[0].content == 'parallel loop gang private(other_var) vector_length(nlon)'
+            assert driver_loops[0].pragma[0].content in (
+                'parallel loop gang private(other_var, more_var) vector_length(nlon)',
+                'parallel loop gang private(more_var, other_var) vector_length(nlon)'
```

Comment:
[no action] Curious, why would the ordering go off here? Should this not be deterministic?

Reply:
Well spotted! 🦅 - 👁️ 
I was wondering the same and I suspect it is here:
https://github.com/ecmwf-ifs/loki/blob/7f7337ef5515edb8b2ddeb9a92ebcfd36852c181/loki/transformations/single_column/annotate.py#L294
We could replace this by
```py
 private_arrays = ', '.join(dict.from_keys(v.name for v in arrays if not v.name_parts[0].lower() in acc_vars))
```
to have a stable sort but unique list. But didn't want to piggy-back this in this hotfix PR

Reply:
Ahhh yes. I think I've fixed this in one of my dev branches. Not urgent now, this will happen one way or another, I think.